### PR TITLE
fix: show more nearby stops that are not part of station

### PR DESCRIPTION
**Pull request by nurmAV**
## Proposed Changes

  - Improves the nearby stops view by listing more stops that are not part of any station
  - Related OTP PR: https://github.com/opentripplanner/OpenTripPlanner/pull/5390

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build
  - Code coverage does not decrease (unless measured incorrectly)

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

### CODE REVIEW

File: app/component/StopNearYou.js [hunk 1]
```diff
@@ -28,7 +28,8 @@ const StopNearYou = (
     }
   }, [currentTime, currentMode]);
   const description = desc || stop.desc;
-  const isStation = !!stop.parentStation || !!stopId;
+  const isStation =
+    stop.locationType === 'STATION' || !!stop.parentStation || !!stopId;
```
Comment (optionsome) [inline app/component/StopNearYou.js]: I think we can probably just simply check `stop.locationType === 'STATION'` here and if the locationType is missing from query, we can add it to there.

File: app/component/StopsNearYouPage.js [hunk 2]
```diff
@@ -213,7 +213,7 @@ class StopsNearYouPage extends React.Component {
 
   getQueryVariables = mode => {
     const { searchPosition } = this.state;
-    let placeTypes = 'STOP';
+    let placeTypes = ['STOP', 'STOP_OR_STATION'];
```
Comment (optionsome) [inline app/component/StopsNearYouPage.js]: I think we probably should only have the 'STOP_OR_STATION' filter here.

File: app/component/StopNearYouContainer.js [hunk 3]
```diff
@@ -126,6 +127,11 @@ const containerComponent = createRefetchContainer(
         ...StopNearYouContainer_stop
         @arguments(startTime: $startTime, omitNonPickups: $omitNonPickups)
       }
+
+      station(id: $stopId) {
```
Comment (optionsome) [inline app/component/StopNearYouContainer.js:85]: I didn't test this yet but does the code somehow magically already handle that sometimes stations is now returned?

**Review by optionsome — CHANGES_REQUESTED** [review]

File: app/component/StopNearYouContainer.js (hunk 3 above)
Comment (nurmAV) [reply app/component/StopNearYouContainer.js:85]: Yes, it seems that the code already handles this. All the stops and stations seem to work like before without changes to code

**Review by optionsome — APPROVED** [review]

**Pull request merged**